src/apis/study-room/exit-study-room.ts

```diff
@@ -0,0 +1,10 @@
+import { http } from '../api';
+
+type Param = {
+  studyRoomId: number;
+  participantId: number;
+};
```

Comment:
다른 파일에서는 함수명+Param으로 표현해주고 있어서 그대로 따라주면 좋을 것 같아요!
```suggestion
type ExitStudyRoomParam = {
  studyRoomId: number;
  participantId: number;
};
```

Reply:
근데 현재 구조에서는 하나의 파일당 하나의 함수라서 오히려 Param으로 표현하는 컨벤션으로 가도 괜찮을 것 같기도 하네요~ 사소하지만 Param보다는 Params가 어떨까 싶습니다~

Reply:
전체적으로 api 타입명에 대해 의논해서 컨벤션을 잡도록 해봐야겠습니다!